operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -202,50 +202,17 @@ Verify that the Lustre file system is available from the management cluster.
     - ncn-w001
     - ncn-w002
     - ncn-w003
+    - ncn-w004
```

Comment:
I would suggest leaving the example with just the three worker nodes for consistency with other examples in throughout the docs.

```suggestion
```